client/my-sites/checkout/composite-checkout/components/wp-order-review-line-items.tsx

```diff
@@ -766,13 +766,15 @@ function IntroductoryOfferCallout( {
 
 function DomainDiscountCallout( {
 	product,
+	hasPlanInCart,
 }: {
 	product: ResponseCartProduct;
+	hasPlanInCart: boolean;
 } ): JSX.Element | null {
 	const translate = useTranslate();
 
 	const isFreeBundledDomainRegistration = product.is_bundled && product.item_subtotal_integer === 0;
-	if ( isFreeBundledDomainRegistration && product.is_renewal ) {
+	if ( isFreeBundledDomainRegistration && ! hasPlanInCart ) {
```

Comment:
I'm not sure I follow this change—If someone renews a plan/domain at the same time, shouldn't we still show this instead of "Discount for first year?"

Reply:
Sorry, we made this change while pairing, noticed the issue, and I hadn't pushed the fix. I think aa99a3c fixes this.